inference-engine/src/inference_engine/cnn_network_ngraph_impl.hpp

```diff
@@ -91,6 +96,8 @@ class INFERENCE_ENGINE_API_CLASS(CNNNetworkNGraphImpl): public ICNNNetwork {
     InferenceEngine::InputsDataMap _inputData;
     std::map<std::string, DataPtr> _outputData;
     const std::vector<IExtensionPtr> _ie_extensions;
+    std::unordered_map<std::string, std::string> _opNames;
+    std::unordered_map<std::string, std::string> _tensorNames;
```

Comment:
At the current moment no, because we don't use framework names in order to get data (we convert new names to legacy). And transformations already propagate legacy names.

In future yes we have to copy tensor names in transformations.